source/_components/sensor.rmvtransport.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "RMV"
+description: "Instructions on how to integrate Rhein-Main public transport departure times into Home Assistant."
+date: 2018-08-02 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rmv.png
+ha_category: Transport
+ha_release: 0.76
+ha_iot_class: "Cloud Polling"
+---
+
+The `rvmtransport` sensor will give you the departure time of the next bus, tram, subway, or train at the next station or stop in the Rhein-Main area public transport network. Additional details such as the line number and destination are present in the attributes.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rmvtransport
+    nextdeparture:
+     - station: STATION_OR_STOP_ID
+```
+
+Configuration variables:
```

Comment:
Please use our configuration tag syntax instead to describe the config options.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration